quarkus/README.md

```diff
@@ -32,8 +32,11 @@ The distribution packages (ZIP and TAR) should be available at [../distribution/
 
 ## Running
 
-    java -jar server/target/lib/quarkus-run.jar
-    
+By default, the HTTP port is disabled and you need to provide the key material to configure HTTPS. If you want to enable
+the HTTP port, run the server in development mode as follows:
+
+    java -Dquarkus.profile=dev -jar server/target/lib/quarkus-run.jar
```

Comment:
It is not, will fix it. It should be `kc.profile=dev`. 

Another point of discussion is if we should just go for `profile={value}` instead of `profile.dev`. The reason being it should just be like any other property.

Reply:
+1 for "kc.profile=dev"